orc8r/protos/BUILD.bazel

```diff
@@ -69,10 +69,15 @@ proto_library(
     srcs = ["redis.proto"],
 )
 
+cpp_proto_library(
+    name = "mconfigs_cpp_proto",
+    protos = [":mconfigs_proto"],
+    deps = [":common_cpp_proto"],
+)
+
 proto_library(
     name = "mconfigs_proto",
     srcs = ["mconfig/mconfigs.proto"],
-    strip_import_prefix = "mconfig/",
```

Comment:
I had to remove this to get the `#include` to work. It's now analogous to the mconfig settings in lte.

Reply:
Yeah I wonder why I added this in the first place :o If the rest of C/C++ services still build fine, this is OK :)